var/spack/repos/builtin/packages/cgal/package.py

```diff
@@ -44,13 +44,29 @@ class Cgal(Package):
     variant('debug', default=False,
             description='Builds a debug version of the libraries')
 
+    # See "7 CGAL Libraries" at:
+    # http://doc.cgal.org/latest/Manual/installation.html
+    variant('core', default=False,
+            description='Build the CORE library for algebraic numbers')
+    variant('imageio', default=False,
+            description='Build utilities to read/write image files')
+    variant('demos', default=False,
+            description='Build CGAL demos')
+
+
     # Essential Third Party Libraries
-    depends_on('boost')
+    depends_on('boost+thread+system')
     depends_on('gmp')
     depends_on('mpfr')
+
+    # Required for CGAL_ImageIO
+    depends_on('opengl', when='+imageio')
```

Comment:
Can you comment this out for now? Otherwise I imagine `spack install cgal` will crash.